lib/siftsciex/decision.ex

```diff
@@ -54,4 +99,13 @@ defmodule Siftsciex.Decision do
 
     time
   end
+
+  defp request_url(entity_id, entity_type, account_id) do
+    "#{base_url()}/#{account_id}/#{entity_type}s/#{entity_id}/decisions"
+  end
+
+  # defp base_url, do: Application.get_env(:siftsciex, :score_url)
```

Comment:
I think this line should be removed

Reply:
Yes, thanks for catching that. I at least got the other one that I decorated with `# DEREK REMOVE THIS WHEN YOU'RE DONE` :joy: